Update bookings-faq.md (#3688)
MicrosoftDocs/microsoft-365-docs: microsoft-365/bookings/bookings-faq.md

Your customers will only see the Bookings calendar that you publish online for the services, times, and staff that you choose to register.

> ### How to change bookable attribute under Manage staff?

Reviewer: ```suggestion
### How do I change a bookable attribute under Manage staff?
```

> When the staff was added, he would have received an email for accepting or rejecting the request. They can open the same email and select "Reject" . This will make him non bookable. We currently do not have a way to toggle this from inside the app.

Reviewer: ```suggestion
When staff members were added, they would have received an email to accept or reject the request. They can open the same email and select "Reject" to make them non-bookable. We currently do not have a way to toggle this from inside the app.
```
